syscall/unistd.c

```diff
@@ -145,6 +144,34 @@ int oe_nanosleep(struct oe_timespec* req, struct oe_timespec* rem)
     return ret;
 }
 
+int oe_clock_gettime(int clock_id, struct oe_timespec* cur_time)
+{
+    int ret = 0;
+    oe_syscall_clock_gettime_ocall(&ret, clock_id, cur_time);
```

Comment:
For consistency with ocalls elsewhere, it would be better to assert that the ocall succeeded via `OE_CHECK(oe_syscall_clock_gettime_ocall(&ret, clock_id, cur_time) == OE_OK);`.